pkg/cmd/pr/create/create.go

```diff
@@ -592,181 +659,282 @@ func NewCreateContext(opts *CreateOptions) (*CreateContext, error) {
 		return nil, err
 	}
 
-	// Resolve target head branch name from either
-	// --head or the current branch.
-	var targetHeadBranch string
-	var targetHeadRepoOwner string
+	// Ignoring the error here is ok, it only means we didn't find a remote.
+	baseRemote, _ := resolvedRemotes.RemoteForRepo(baseRepo)
 
-	promptForHeadRepo := true
+	// This closure provides an easy way to instantiate a CreateContext with everything other than
+	// the refs. This probably indicates that CreateContext could do with some rework, but the refactor
+	// to introduce PRRefs is already large enough.
+	var newCreateContext = func(refs refs) *CreateContext {
+		baseTrackingBranch := refs.BaseRef()
+		if baseRemote != nil {
+			baseTrackingBranch = fmt.Sprintf("%s/%s", baseRemote.Name, baseTrackingBranch)
+		}
```

Comment:
TODO: write something to explain this. Also probably move the base remote lookup into the closure.